packages/discovery-provider/integration_tests/tasks/entity_manager/test_comment.py

```diff
@@ -30,6 +30,15 @@
         {"user_id": 1, "wallet": "user1wallet"},
         {"user_id": 2, "wallet": "user2wallet"},
         {"user_id": 3, "wallet": "user3wallet"},
+        {"user_id": 4, "wallet": "user4wallet"},
```

Comment:
Mind not adding this to the root entities for perf/readability reasons? You can merge in those users in your tests, check some other examples